lib/rack-cas/session_store/redis.rb

```diff
@@ -0,0 +1,102 @@
+module RackCAS
+  module RedisStore
+    class Session
+      @@client = Redis.new(path: '/tmp/redis.sock',driver: :hiredis)
```

Comment:
I'm a bit concerned with the hard-coded path and driver here. What do you think about adding them to [`RackCAS::Configuration`](https://github.com/biola/rack-cas/blob/master/lib/rack-cas/configuration.rb)?